docs/source/Advanced-Usage.md

```diff
@@ -124,7 +130,11 @@ runtime_configuration = {
 }
 pyintelowl_client.send_file_analysis_request(..., runtime_configuration=runtime_configuration)
 ```
+#### PhoneInfoga
+PhoneInfoga provide several [scanners](https://sundowndev.github.io/phoneinfoga/getting-started/scanners/) to extract as much information as possible from a given phone number. Those scanners may require authentication, so they're automatically skipped when no authentication credentials are found.
 
+By default the scanner used is `local`. The analyzer will automatically parse API keys saved as environment variables present in the `docker/env_file_integrations` file.
```

Comment:
can you provide an example for other scanners? to see if they work as intended

Reply:
do you mean that i should provide screenshot for other scanners?

Reply:
this?
![image](https://github.com/intelowlproject/IntelOwl/assets/102874321/dc597a25-41c0-4b28-9d04-f0ae868b8c76)

Reply:
ty, cool